docs/getting-started-scalar-manager.md

```diff
@@ -0,0 +1,120 @@
+# Getting Started with Helm Charts (Scalar Manager)
+Scalar Manager is a web-based dashboard that allows users to:
+* check the health of the Scalar products
+* pause and unpause the Scalar products to backup or restore underlying databases
+* check the metrics and logs of the Scalar products through Grafana dashboards
+The users can pause or unpause Scalar products through Scalar Manager to backup or restore the underlying databases.  
```

Comment:
```suggestion
* check the metrics and logs of the Scalar products through Grafana dashboards

The users can pause or unpause Scalar products through Scalar Manager to backup or restore the underlying databases.  
```
I think the `new line` is necessary to separate the bullets and the next new sentence in the markdown syntax.